contrib/devtools/fix-copyright-headers.py

```diff
@@ -16,7 +16,7 @@
 import re
 
 year = time.gmtime()[0]
-CMD_GIT_DATE = 'git log --format=@%%at -1 %s | date +"%%Y" -u -f -'
+CMD_GIT_DATE = 'git log --format=%%ad --date=short -1 %s | cut -d"-" -f 1'
```

Comment:
> LazyTM and ItWorksTM, also you need to call at least git to make this script work. 

You don't need to call it through the shell though. Popen, by default, just invokes the process directly. See e.g. how `github-merge.py` interacts with git.